pom.xml

```diff
@@ -47,46 +47,46 @@ THE SOFTWARE.
-        <annotation-indexer.version>1.14</annotation-indexer.version>
         <asm.version>9.1</asm.version>
         <byte-buddy.version>1.9.10</byte-buddy.version>
-        <caffeine.version>2.9.2</caffeine.version>
         <cobertura.version>1.12.1</cobertura.version>
         <commons-csv.version>1.8</commons-csv.version>
         <commons-lang3.version>3.11</commons-lang3.version>
         <configuration-as-code.version>1.55.1</configuration-as-code.version>
         <credentials.version>2.6.1.1</credentials.version>
         <doxia-sink-api.version>1.1.2</doxia-sink-api.version>
-        <gson.version>2.8.9</gson.version>
+        <gson.version>2.8.8</gson.version>
         <hpi.compatibleSinceVersion>3.0</hpi.compatibleSinceVersion>
         <influxdb-client-java.version>4.1.0</influxdb-client-java.version>
         <jacoco.version>1.0.12</jacoco.version>
         <java.level>8</java.level>
         <jenkins.version>2.289.1</jenkins.version>
         <jimfs.version>1.0</jimfs.version>
         <junit.version>4.13.1</junit.version>
-        <junit_plugin.version>1.24</junit_plugin.version>
-        <kotlin-stdlib.version>1.6.0</kotlin-stdlib.version>
-        <mailer.version>391.ve4a_38c1b_cf4b_</mailer.version>
+        <kotlin-stdlib.version>1.3.71</kotlin-stdlib.version>
+        <mailer.version>1.20</mailer.version>
         <maven-reporting-api.version>3.0</maven-reporting-api.version>
         <metrics.version>4.0.2.8.1</metrics.version>
-        <mockito-core.version>2.26.0</mockito-core.version>
+        <mockito-core.version>4.4.0</mockito-core.version>
         <performance.version>3.0</performance.version>
         <perfpublisher.version>8.05</perfpublisher.version>
-        <plexus-utils.version>3.0.16</plexus-utils.version>
+        <plexus-utils.version>1.5.12</plexus-utils.version>
         <plexus-container-default.version>1.0-alpha-30</plexus-container-default.version>
         <plain-credentials.version>1.7</plain-credentials.version>
         <robot.version>3.0.0</robot.version>
-        <script-security.version>1.75</script-security.version>
-        <slf4j.version>1.7.30</slf4j.version>
-        <spotbugs-annotations.version>4.2.0</spotbugs-annotations.version>
+        <script-security.version>1.58</script-security.version>
         <spotbugs.failOnError>false</spotbugs.failOnError>
-        <structs.version>1.23</structs.version>
+        <spotbugs.version>4.2.0</spotbugs.version>
+        <structs.version>308.v852b473a2b8c</structs.version>
         <symbol-annotation.version>1.10</symbol-annotation.version>
         <workflow-api.version>2.33</workflow-api.version>
         <workflow-step-api.version>2.20</workflow-step-api.version>
         <workflow-durable-task-step.version>2.35</workflow-durable-task-step.version>
+        <scm-api.version>2.2.7</scm-api.version>
+        <annotation-indexer.version>1.14</annotation-indexer.version>
```

Comment:
I would like to keep these in alphabetical order, so that they're easy to locate when I need to bump versions again.